src/app/helpers/users_helper.rb

```diff
@@ -2,4 +2,8 @@ module UsersHelper
   def user_image_url(user)
     "https://avatars0.githubusercontent.com/u/#{user.github_id}?v=4"
   end
+
+  def user_github_account_url(user)
+    "https://github.com/#{user.username}"
```

Comment:
i don't know what risk could raised for a non filter input.

Reply:
the same in `user_image_url`
@emad-elsaid